### DISCUSSION TITLE
Killcurly isn't working...

### DISCUSSION THREAD

**Discussion by rabbitmurias** [3kh0/ext-remover#530] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I followed the steps on the page for performing killcurly but all websites are still blocked by GG .-.

Also, I did look for similar discussions but I saw that others had it work. I'm not sure what's going on here.

**Comment by Aka-but-nice** [comment]
It was confirmed not to work with GG

**Reply by rabbitmurias** [reply]
Oh, thanks for letting me know.
